src/daemon/plugin.c

```diff
@@ -1085,48 +1091,78 @@ static int plugin_compare_read_func(const void *arg0, const void *arg1) {
  * The AlignRead value is in seconds. The value modulo 60 determines the second
  * of a minute. The integer part of the value divided by 60 determines the
  * minute of an hour. Optionally, decimal places can be used to set the
- * fraction of a second (up to milliseconds precision).
+ * fraction of a second (up to microseconds precision).
```

Comment:
Do you know something about schedulers precision (at OS level)?
Are you seriously thinking OS assigns microseconds quantums to tasks??

Want to disappoint you: quantum has milliseconds dimension.

Reply:
This is only the internally used precision from the timeval structure. I can extend the comment to make clear that the alignment will very probable not be accurate to microseconds.

Reply:
I think `(up to microseconds precision)` just needs to be removed.

Reply:
Done.